### DISCUSSION TITLE
Docs about getInitialNamespaces and RemixI18Next

### DISCUSSION THREAD

**Discussion by miradontsoa** [sergiodxa/remix-i18next#178] [General]
Hello,
I think, there is TypeScript issue with in the current version v6, the TypeScript import of `RemixI18Next `and `getInitialNamespaces `are 
`import { RemixI18Next } from "remix-i18next/build/server";`
and 
`import { getInitialNamespaces } from "remix-i18next/build/client";`
not `import { RemixI18Next } from "remix-i18next/server";` and` import { getInitialNamespaces } from "remix-i18next/client";` as mentionned in the doc but when building the project, `"remix-i18next/server";` and  `"remix-i18next/client";` are the right one.

**Comment by sergiodxa** [comment]
Ensure you have moduleResolution: “bundler” in your tsconfig.json, new Remix app come with all but old one doesn’t

You may need to upgrade to TS v5

**Reply by miradontsoa** [reply]
Thanks, moduleResolution was "node" with the blues-stack template (may be they should change it) so changing it to bundler fixed it.
